### ISSUE TITLE
single selection datagrid got  wrong when assignd a data to select first line

### ISSUE THREAD

**Issue by pengpengshui** [vmware/clarity#2116]
**Select one ...**  (check one with "x")
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior
I use clarity to build a single selection datagrid. when complete loading the component , the first line shoud be selected.

### Actual behavior
when I got the first data from database and assigned to a the specific variable. the console of browser  call back a wrong tips. as a consequence, the program got confusion

### Reproduction of behavior
this are part of codes about .html and .ts file
```html
<clr-datagrid [clrDgLoading]="loading"  [(clrDgSingleSelected)]="selectedRow" [clDgRowSelection]="true">
```
```typescript
retrieveRules(ruleName: string = ''): void {
    this.loading = true;
    this.selectedRow = null;
    toPromise<ReplicationRule[]>(this.replicationService
      .getReplicationRules(this.projectId, ruleName))
      .then(rules => {
        this.changedRules = this.rules;
        this.selectedRow = this.changedRules[0];
        this.loading = false;
      }
      ).catch(error => {
        this.errorHandler.error(error);
        this.loading = false;
      });
  }
```
the issue casued by ` this.selectedRow = this.changedRules[0]` 

the console error:
![1522306662 1](https://user-images.githubusercontent.com/29880301/38076914-b4fe7b7c-3369-11e8-8056-213df61264ff.jpg)

when I replace `clarity-angular`, `clarity-icons`,`clariy-ui`  three package's version,  from ^0.10.26  to 0.10.23. the bug is gone .

### Environment details
my package.json file:
![1522310230 1](https://user-images.githubusercontent.com/29880301/38077002-011cdb8e-336a-11e8-9bff-9630497dc7cb.jpg)

* **OS and version:**  Win10

* **Browser:** chrome

**Comment by youdz** [comment]
This is a duplicate of #2002, which was fixed in `0.11.10`. Can you please try upgrading to the latest version of Clarity to check that it solves your problem?

**Comment by pengpengshui** [comment]
I see clarity offical website  [Project Clarity](https://vmware.github.io/clarity/documentation/v0.11/get-started), the latest version is 0.11.6,   if the latest version  resolved  the problem?

**Comment by SayakMukhopadhyay** [comment]
The latest is `0.11.11`
![image](https://user-images.githubusercontent.com/13748882/38134537-37a76ac6-3431-11e8-969b-370fb8623db9.png)

So it should be fixed in `0.11.`0` as mentioned by youdz

**Comment by gnomeontherun** [comment]
@pengpengshui you must be looking at the sketch template version number.
